lib/fog/kubevirt/compute/requests/get_vnc_console_details.rb

```diff
@@ -0,0 +1,21 @@
+module Fog
+  module Kubevirt
+    class Compute
+      class Real
+        def get_vnc_console_details(name, namespace)
+            {
+             :host => @host,
+             :port => @port,
+             :path => "/apis/subresources.kubevirt.io/v1alpha3/namespaces/#{namespace}/virtualmachineinstances/#{name}/vnc",
+             :token => @opts[:auth_options][:bearer_token]
+            }
```

Comment:
no need for certificate in this context ? where can I see the code that uses this method ?

Reply:
AFIK, there is no need in the certificates, only the issuer id should by recognise by the browser.
this is PR the uses the method:
https://github.com/theforeman/foreman_kubevirt/pull/111
this is the PR for novnc:
https://github.com/novnc/noVNC/pull/1262
and this one for foreman:
https://github.com/theforeman/foreman/pull/6907

Reply:
@masayag 
so about the certificate, I have no options authorized the issuer id in the browser, therefore I don't need the cert file. 
please review again :)

Reply:
@shiramax one thing I'd like to avoid in this context is specifying hard-coded the api version in the url.
Please replace `v1alpha3` with `kubevirt_client.version`